### PR TITLE
Generate Ioym binaries

### PR DESCRIPTION
**Pull request by gleon99**

### CODE REVIEW

File: ioym/src/main.rs [hunk 1]
```diff
@@ -117,7 +117,7 @@ where
             let available = match r.fill_buf() {
                 Ok(n) => n,
                 Err(ref e) if e.kind() == std::io::ErrorKind::Interrupted => continue,
-                Err(e) => Err(e)?,
+                Err(e) => return Err(e.into()),
```
Comment (drrlvn) [inline ioym/src/main.rs]: Why? The advantage of `?` is that it calls `into()`.

Comment (gleon99) [reply ioym/src/main.rs]: Indeed, though the disadvantage is having a failed pipeline:
https://rust-lang.github.io/rust-clippy/master/index.html#try_err

**Comment by drrlvn** [comment]
@gleon99 Looks good, resolve conflicts and we can merge.

**Comment by drrlvn** [comment]
@gleon99 ping

**Comment by gleon99** [comment]
All conflicts resolved.

**Pull request merged**